transaction-gossip.md

```diff
@@ -65,14 +54,30 @@ proof       := TODO
 transaction := TODO
 ```
 
+## Secondary Routing Table
+
+Clients in the Transaction Gossip Network are expected to maintain both the standard routing table and a secondary routing table that is anchored to node radius values.
+
+The secondary routing table is subject to all of the same maintenance and management rules as the primary table.  Any node that is added to the secondary routing table must also satisfy the following validity condition:
+
+```
+node.transaction_radius >= distance(node.node_id, self.node_id)
+```
+
+The additional validity rule aims to ensure that the table is populated with nodes will have shared interest in mostly the same transactions as us.
+
 
 ## Gossip Algorithm
 
-The gossip mechanism for transactions is designed to allow nodes to decide how much of the transaction pool they wish to process.
+The gossip mechanism for transactions is designed to allow DHT nodes to control what percentage of the transaction pool they wish to process.
 
 ### Radius
 
-All DHT nodes in the network will maintain a `radius` which communicates how much of the transaction pool they process.  This value is a 256 bit integer.  A DHT node is expected to process transactions for which `distance(node_id, content_id) <= radius`.
+We use the term "radius" to refer to the mechanism through which a node may limit how much of the transaction pool they wish to process.  The `radius` is a 256 bit integer.  
+
+A DHT node that wishes to process the full transaction pool would publis a radius value of `2**256-1`. We refer to such a DHT node as a "full radius node".
```

Comment:
It could be more user friendly to use a normalized radius instead. Normalizing to 1 would be easy to understand. It is hard to make sense of any number that is kind of in the middle of 2*256 and 1 without resorting to a calculator. 

If a more log scale is preferred instead of linear, it could be log and then normalized as well.